packages/flutter/lib/src/painting/binding.dart

```diff
@@ -70,8 +70,15 @@ mixin PaintingBinding on BindingBase, ServicesBinding {
   ImageCache createImageCache() => ImageCache();
 
   /// Calls through to [dart:ui] with [decodedCacheRatioCap] from [ImageCache].
-  Future<ui.Codec> instantiateImageCodec(Uint8List list) {
-    return ui.instantiateImageCodec(list);
+  Future<ui.Codec> instantiateImageCodec(Uint8List list, {
+    int cacheWidth,
+    int cacheHeight,
```

Comment:
when you add arguments, you should document them. :-)

Reply:
(especially here where the semantics are different than what you're forwarding to)